src/applications/debt-letters/components/DebtCardsListV2.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import DebtLetterCardV2 from './DebtLetterCardV2';
+import { Link } from 'react-router';
+
+const DebtCardsListV2 = ({ debts, isError }) => {
+  const renderAlert = () => (
+    <div
+      className="usa-alert usa-alert-error vads-u-margin-top--0 vads-u-padding--3"
+      role="alert"
+    >
+      <div className="usa-alert-body">
+        <h3 className="usa-alert-heading">
+          Information about your current debts is unavailable.
+        </h3>
+        <p className="vads-u-font-family--sans">
+          You can't view information about your current debts because something
+          went wrong on our end.
+        </p>
+        <p className="vads-u-margin-bottom--1">
+          <strong>What you can do</strong>
+        </p>
+        <p className="vads-u-font-family--sans vads-u-margin-y--0">
+          You're still able to download your debt letters from the list below.
+          If you need help resolving a debt, or you would like to get
+          information about a debt that has been resolved, call the Debt
+          Management Center at{' '}
+          <a href="tel: 800-827-0648" aria-label="8 0 0. 8 2 7. 0 6 4 8.">
+            800-827-0648
+          </a>
+          {'.'}
+        </p>
+      </div>
+    </div>
+  );
+
+  return (
+    <>
+      <h2
+        id="currentDebts"
+        className="vads-u-margin-top--4 vads-u-margin-bottom--2"
+      >
+        Current debts
+      </h2>
+      {isError && renderAlert()}
+      {!isError &&
```

Comment:
do we need to check `isError` again here since were already calling renderAlert above?

Reply:
We do because both will be called since its inside the return

Reply:
right i was thinking of something like returning using a ternary but short circuiting works too

Reply:
I would love to return early but I need the content above and below too :-/